Add interfaces/css-regions.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/css-regions.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166

gsnedders commented:
@foolip @stephenmcgruer #21368 removed this, but @autofoolip will just readd it given it appears in a spec…

stephenmcgruer commented:
We need to remove it as a spec reffy cares about. @foolip knows the magic for this, so I will leave it to them to do so.

foolip commented:
This is one of a bunch of open PRs in https://github.com/web-platform-tests/wpt/pulls/autofoolip for specs that nobody is very interested in implementing. I think https://github.com/w3c/browser-specs/ will eventually address this, leaving this open.

autofoolip committed:
Add interfaces/css-regions.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/css-regions.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166
